backend/prisma/schema.prisma

```diff
@@ -0,0 +1,31 @@
+datasource db {
+    url      = env("DB_URL")
+    provider = "postgresql"
+}
+
+generator client {
+    provider = "prisma-client-js"
+}
+
+model Game {
+    id           String   @id @default(cuid())
+    name         String
+    description  String
+    platform     Platform @relation(fields: [platformName], references: [name])
+    platformName String
+    dateReleased DateTime
+    //dateAdded DateTime
+    //canBorrow Boolean
+    //playerMin    Int
+    //playerMax    Int
+    playtime     Int // Stored in minutes
```

Comment:
Shouldn't this be changed so that it's explicitly in minutes, such as renaming it to `playtimeMinutes`?

Reply:
Fixed